tfe/resource_tfe_policy_set.go

```diff
@@ -76,6 +118,19 @@ func resourceTFEPolicySetCreate(d *schema.ResourceData, meta interface{}) error
 		options.Description = tfe.String(desc.(string))
 	}
 
+	if vcsOpts := d.Get("vcs_repo").([]interface{}); len(vcsOpts) > 0 {
+		options.VCSRepo = &tfe.VCSRepoOptions{
+			Branch:            tfe.String(vcsOpts[0].(map[string]interface{})["branch"].(string)),
+			Identifier:        tfe.String(vcsOpts[0].(map[string]interface{})["identifier"].(string)),
+			IngressSubmodules: tfe.Bool(vcsOpts[0].(map[string]interface{})["ingress_submodules"].(bool)),
+			OAuthTokenID:      tfe.String(vcsOpts[0].(map[string]interface{})["oauth_token_id"].(string)),
+		}
+	}
```

Comment:
I think it would be nice to reuse the same logic used [here](https://github.com/terraform-providers/terraform-provider-tfe/blob/master/tfe/resource_tfe_workspace.go#L145-L159). Partly because it does an extra check for the `branch` and also because (IMHO) it looks a little cleaner.